<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserSignup</class>
 <widget class="QWidget" name="UserSignup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(237, 106, 90);</string>
  </property>
  <widget class="QToolButton" name="signupBtn_2">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>491</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
background-color: rgb(155, 193, 188);
border: 1px solid;</string>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::InstantPopup</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="loginBtn_2">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>541</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
background-color: rgb(244, 241, 187);
border: 1px solid;</string>
   </property>
   <property name="text">
    <string>Sign In</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::InstantPopup</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
  <widget class="QLabel" name="logo_label_2">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>80</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/res/logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwd_lineEdit_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>331</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid;
border-radius: 5px;
background-color: rgb(230, 235, 224);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="pwd_label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>361</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmpwd_lineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>451</y>
     <width>331</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid;
border-radius: 5px;
background-color: rgb(230, 235, 224);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_lineEdit_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>329</y>
     <width>331</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid;
border-radius: 5px;
background-color: rgb(230, 235, 224);</string>
   </property>
  </widget>
  <widget class="QLabel" name="confirmpwd_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>422</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Confirm Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="username_label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
